src/samplers/string.js

```diff
@@ -1,9 +1,18 @@
 'use strict';
 
 import { ensureMinLength, toRFCDateTime, uuid } from '../utils';
+import { fromRegExp } from '../vendor/fakerFromRegexp';
```

Comment:
Let's make it a bit more consistent.

let's move it from vendor folder into `samplers/string-regex.js` and rename the internal input from `fromRegExp` to `regexSample`.

Reply:
OK, adjusted. I could not import as just `regexSample` due to naming conflict so I did a namespace import